src/pages/public_registration/action_accordions/DeletePublication.tsx

```diff
@@ -27,13 +29,15 @@ interface DeletePublicationProps {
 }
 
 export const DeletePublication = ({ registration }: DeletePublicationProps) => {
-  const [showDeleteButton, setShowDeleteButton] = useState(false);
+  const [showDeleteField, setShowDeleteField] = useState(false);
   const [showDeleteModal, setShowDeleteModal] = useState(false);
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const [selectedDuplicate, setSelectedDuplicate] = useState<Registration | null>(null);
   const history = useHistory();
 
+  const userCanUnpublish = userCanUnpublishRegistration(registration);
```

Comment:
Tror vi må utvide med en ekstra sjekk her, for `canUnpublish` er sikkert den vi må bruke for å sjekke om bruker har lav til å fjerne publisert fil også? Altså må enten backend legge til en ny `allowedOperation` "Unpublih-File", eller så må vi legge til en ekstra sjekk her. Sikkert lurt å gjøre en avsjekk med @torbjokv / @ketilaa om hva de tenker backend skal tilby her

```suggestion
  const userCanUnpublish = userCanUnpublishRegistration(registration) && !hasPublishedFile;
```

Reply:
Denne sjekken funker her. Men i unpublish file må vi tenke ut noe nytt. Det blir neste PR